src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java

```diff
@@ -265,4 +265,29 @@ public static By locatorOfLoadingSpinnerOnValidateButton() {
   @Deprecated
   @FindBy(how = How.XPATH, using = "//*[@data-cy='plugin-validation-error-msg']")
   public static WebElement pluginValidationErrorMsg;
+
+  @FindBy(how = How.XPATH, using = "//img[@id='resource-center-btn']")
+  public static WebElement uploadFileButton;
+
+  public static By importuploadButton() {
```

Comment:
Rename: importPipelineButton

Reply:
This is not used outside Locators class. Remove this and use the locator in "chooseFile" WebElement

Reply:
Done.